server/lib/OPodSync/API.php

```diff
@@ -273,7 +273,7 @@ public function handleNextCloud(): ?array
 					'token' => $id,
 					'endpoint' => $this->url('index.php/login/v2/poll'),
 				],
-				'login' => $this->url('login?token=' . $id),
+				'login' => $this->url('login.php?token=' . $id),
```

Comment:
when `.php` is missing, we end up on the normal login page (through the fallback resource) instead of the app access login page